C/common/filter_pipeline.cpp

```diff
@@ -312,8 +312,16 @@ bool FilterPipeline::setupFiltersPipeline(void *passToOnwardFilter, void *useFil
 void FilterPipeline::cleanupFilters(const string& categoryName)
 {
 
-	// Shutdown filters - do this down the pipeline to allow
-	// filters to pass data on to the next ask they shutdown
+	// Shutdown filters - do this down the pipeline, starting
+	// from the first filter in the pipeline This allows a filter
```

Comment:
first filter in the pipeline This allows... -> first filter in the pipeline. This allows... (add a period after "pipeline").